x11-wm/2bwm/2bwm-9999.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit savedconfig
```

Comment:
Please use `toolchain-funcs`, in order to set a cc compiler.

Currently, it only calls `cc`
-> cc [..] -c -o 2bwm.o 2bwm.c